modules/rounds.js

```diff
@@ -387,6 +387,16 @@ __private.getOutsiders = function(scope, cb, tx) {
  * @todo Add description for the params and the return value
  */
 __private.sumRound = function(scope, cb, tx) {
+	// When we need to sum round just after genesis block (height: 1)
+	// - set data manually to 0, they will be distributed when actual round 1 is summed
+	if (scope.block.height === 1) {
+		library.logger.debug(`Summing round - ${scope.round} (genesis block)`);
+		scope.roundFees = 0;
+		scope.roundRewards = [0];
+		scope.roundDelegates = [scope.block.generatorPublicKey];
+		return setImmediate(cb);
+	}
+
 	library.logger.debug('Summing round', scope.round);
 
 	(tx || library.db).rounds
```

Comment:
Th eline below states:

```
.summedRound(scope.round, constants.activeDelegates)
```
To avoid any mistake we should always calculate `round` from the `height` of the block given in scope. Because its fully dependent value. 

With this approach there can be mistake in future to sum a `round` with wrong block height.

Reply:
@nazarhussain good refactoring suggestion. I would appreciate if you could open it as a separate issue.